src/templates/documentation-page.jsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable react/no-danger */
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { graphql } from 'gatsby';
+import { I18nextProvider, useTranslation } from 'react-i18next';
+import { Grid, GridContainer } from '@trussworks/react-uswds';
+import { i18next, Layout } from '../components';
+
+import '@trussworks/react-uswds/lib/uswds.css';
```

Comment:
Do these imports have to be repeated for each template? If so, might want to avoid this repetition.

Reply:
Yes otherwise the uswds styles are not applied

Reply:
Can you think of a way to avoid repetition? We can, of course, put in the documentation a proposed template for templates that would include these imports. However, is there a way we can avoid repeating?

Reply:
Yeah my only thought was a template/layout component for templates that includes all the imports

Reply:
Sounds good!